effects/sleep.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import time
+
+import click
+
+from rpi_ws281x import PixelStrip, Color
+
+# LED strip default configuration:
+#LED_COUNT = 50       # Number of LED pixels.
+LED_PIN = 18          # GPIO pin connected to the pixels (18 uses PWM!).
+LED_FREQ_HZ = 800000  # LED signal frequency in hertz (usually 800khz)
+LED_DMA = 10          # DMA channel to use for generating signal (try 10)
+LED_BRIGHTNESS = 255  # Set to 0 for darkest and 255 for brightest
+LED_INVERT = False    # True to invert the signal (when using NPN transistor level shift)
+LED_CHANNEL = 0       # set to '1' for GPIOs 13, 19, 41, 45 or 53
```

Comment:
This was not the intention. This constants comes from the original examples and are used in the PixelStrip class. The most of them would not change anyway

LED_PIN - There we don´t have a lot of choice ... i even don´t know if other pins work ... may be it is different for RPi4
LED_FREQ_HZ - Here we also do not have much choice. If your using a slow strip based on WS2812B then you need to change the frequency to match your LED Strip. But with WS2812B you can not chain up that much of LEDs without dropping your FPS to much.
LED_DMA - Don´t know what this value is configuring xD
LED_BRIGHTNES - I would not recommend to change this value ... use RGB value instead ... otherwise it seems that if you reduce the global brightness you will have strange behavior on the effects
LED_INVERT - This is about the  hardware ... i don´t expect that you will have a need to invert the signal
LED_CHANNEL - If you change the LED_PIN then you probably need to change also this flag ... but don´t want this as a parameter i think xD

I think all this parameters we will never change, so don´t wand them as a parameter for script execution. This will not give us any values but make the understanding of the parameters much harder. If in the future we discover that we also need to change some of this parameters, we then can still adjust the scripts